content/chainguard/chainguard-images/authenticating.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Authenticating to Chainguard Registry"
+type: "article"
+description: "A guide on authenticating to the Chainguard Registry to get images"
+date: 2023-03-21T15:10:16+00:00
+lastmod: 2023-03-21T15:10:16+00:00
+draft: false
+images: []
+menu:
+  docs:
+    parent: "chainguard-images"
+weight: 500
+toc: true
+---
+
+### Public Images
+
+All public Chainguard Images are available for free.
+
+However, logging in with a Chainguard account and authenticating when pulling from the registry can provide a mechanism for Chainguard to contact you if there are any issues with images you are pulling.
+
+This may include upcoming deprecations or changes in behavior, or critical vulnerabilities found and remediations applied to images being pulled.
+
+### Authenticating with the credential helper
+
+The easiest way to configure authentication is to use the credential helper included with `chainctl`.
+
+First install `chainctl`, then log in and configure the credential helper:
+
+```
+chainctl auth login
+chainctl auth configure-docker
+```
+
+This will update your Docker config file to call `chainctl` when an auth token is needed. A browser window will open when the token needs to be refreshed.
+
+Pulls authenticated in this way are associated with your user.
+
+### Authenticating with a Pull Token
+
+You can also create a "pull token" using `chainctl`.
+
+First install `chainctl`, then log in and configure a pull token:
+
+```
+chainctl auth login
+chainctl auth configure-docker --pull-token
```

Comment:
Perhaps explicitly include the `--ttl` flag, so it is very in your face that this is a time-bombed credential?